frontend/lib/about/about_page.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:package_info/package_info.dart';
+
+import 'content_tile.dart';
+import 'texts.dart';
+
+class AboutPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+        child: FutureBuilder<PackageInfo>(
+            future: PackageInfo.fromPlatform(),
+            builder: (context, snapshot) {
+              List<Widget> children;
+              if (snapshot.hasData) {
+                children = [
+                  Center(
+                      child: Padding(
+                          padding: EdgeInsets.all(10),
+                          child: ClipRRect(
+                              borderRadius: BorderRadius.circular(20.0),
+                              child: Image(
+                                image: AssetImage("assets/icon/icon.png"),
+                                height: 100.0,
+                                width: 100.0,
+                                fit: BoxFit.cover,
+                              )))),
+                  Center(
+                      child: Text(snapshot.data.appName,
+                          style: Theme.of(context).textTheme.headline5)),
+                  Center(
+                      child: Text(snapshot.data.version,
+                          style: Theme.of(context).textTheme.bodyText2)),
+                  const Divider(
+                    height: 20,
+                    thickness: 1,
+                  ),
+                  InkWell(
+                    child: Column(
+                      children: [
+                        Center(
+                            child: Text("Herausgeber",
+                                style: Theme.of(context).textTheme.subtitle2)),
+                        Padding(
+                            padding: EdgeInsets.all(10),
+                            child: Text(publisherAdress,
+                                style: Theme.of(context).textTheme.bodyText1)),
+                      ],
+                    ),
+                    onTap: () {
+                      Navigator.push(
+                          context,
+                          MaterialPageRoute(
+                            builder: (context) => ContentPage(
+                                title: "Herausgeber", content: Text(publisher)),
+                          ));
+                    },
```

Comment:
I did not notice (until I saw the code), that I could click on Herausgeber. Maybe a button (or a link) with "Mehr Informationen" would help here.

Also there seems to be more padding to below the Herausgeber section than above. Probably it looks best, if there's an equal amount of padding.